bb/tasks.clj

```diff
@@ -0,0 +1,35 @@
+(ns tasks
+  (:require [babashka.deps :as deps]
+            [babashka.fs :as fs]
+            [babashka.process :as p]
+            [clojure.string :as str]))
+
+(defn compile-native
+  "Compile library to standalone jar and a native executable program."
+  []
+  (let [graalvm-home (or (System/getenv "GRAALVM_HOME")
+                         (throw (Exception. "Please set GRAALVM_HOME.")))
+        java-home (str (fs/path graalvm-home "bin"))
+        deps-clj-version (slurp "resources/DEPS_CLJ_VERSION")]
+    (println "Building deps " deps-clj-version)
+    (p/shell "lein" "deps.clj" "-Spath" "-Sdeps" "{:deps {borkdude/deps.clj {:mvn/version \"0.0.1\"}}}")
+    (p/shell "lein with-profiles +native-image do clean, uberjar")
+    (let [native-image (str (fs/path graalvm-home "bin"
```

Comment:
This is no longer necessary in newer versions of bb. So if we upgrade bb in circleci and appveyor, we can simplify this code.

Reply:
Sorry @borkdude, it is not clear to me which line(s) you are referring to (it displays from line n14 to n15). The line I don't understand its meaning is line n.15, and it appears again  at the end of the fn which I though it was only there for testing the native executable that it works. Thanks

Reply:
Ah. I mean the test if we're in Windows or not. With `p/shell` you no longer have to specify the extension in Windows.

Reply:
> Ah. I mean the test if we're in Windows or not. With `p/shell` you no longer have to specify the extension in Windows.

Ah, this must be something very new, I seem to remember (I could be wrong) the extension was added only when the program was without a path component.

Reply:
Ah I see! Yes, you're right, I didn't see that.

Reply:
Is there something more you wanted me to look at here?